requirements-base.txt

```diff
@@ -67,8 +67,8 @@ toronado==0.1.0
 typing-extensions==3.10.0.2
 ua-parser==0.10.0
 unidiff==0.7.1
-urllib3[brotli]==1.25.11
-brotlipy==0.7.0
+urllib3[brotli]==1.26.8
```

Comment:
Can we bump to 1.26.9? 😄

Reply:
Yeah. Thanks for letting me know, forgot about this. Now that https://github.com/getsentry/sentry/pull/34524 is done, we can do this.